flutter/news/lib/features/frontpage/presentation/bloc/top_headlines_state.dart

```diff
@@ -5,10 +5,15 @@ part 'top_headlines_state.freezed.dart';
 
 @freezed
 class ArticlesState with _$ArticlesState {
-  const factory ArticlesState.initial() = _Initial;
-  const factory ArticlesState.loading() = _Loading;
+  const factory ArticlesState.initial() = Initial;
```

Comment:
Where do you need this as `public`?

I don't think there is a case where we need `_Loading`, `_Initial`, `_Error` to be public, right? it should be enough if we deal with `state.when()` when having to see which type is which one

Reply:
When i added a message to the Error one, i had to change the way i did the test, in the test i am now only checking for the Type (isA<Error>()) so i needed it to be public, and i thought they all should be for the same reason